sdk/core/Azure.Core/src/HttpRange.cs

```diff
@@ -26,22 +26,22 @@ namespace Azure
         /// Gets the size of the <see cref="HttpRange"/>.  null means the range
         /// extends all the way to the end.
         /// </summary>
-        public long? Count { get; }
+        public long? Length { get; }
 
         /// <summary>
         /// Creates an instance of HttpRange
         /// </summary>
         /// <param name="offset">null means offset is 0</param>
-        /// <param name="count">null means to the end</param>
-        public HttpRange(long? offset = default, long? count = default)
+        /// <param name="length">null means to the end</param>
+        public HttpRange(long? offset = default, long? length = default)
```

Comment:
I get why `length` is nullable, but why offset? It could just default to 0 with the same effect.